common/apm/src/metrics/network.rs

```diff
@@ -0,0 +1,82 @@
+use super::{
```

Comment:
Don't use `super` and where is the timeout metric for `RPC`?

Reply:
> Don't use `super` and where is the timeout metric for `RPC`?

Done